UPGRADING.md

```diff
@@ -0,0 +1,26 @@
+# Upgrading Bothan deployments
+
+## 2017-08-16
+
+Changes to dependencies mean that some new system dependencies are required for deployments created before date. `cmake` now must be installed on systems hosting Bothan instances.
+
+### Heroku deployments
+
+If you have deployed using [deploy.bothan.io](https://deploy.bothan.io) or hosted your own instance on Heroku, you will need to make the following manual changes.
+
+* Log in to your Heroku dashboard, and visit the settings page for your Bothan app.
+* Press the "upgrade stack" button to use the new `heroku-16` stack instead of `cedar-14`.
+* In the buildpacks section, add the following entries:
+  * https://github.com/100yrs/heroku-buildpack-cmake.git
+  * https://github.com/heroku/heroku-buildpack-google-chrome.git
+* Remove the `heroku/ruby` buildpack and re-add it so it is at the BOTTOM of the list.
+
+#### Redeploying
+
+If your app is automatically deploying from a GitHub repository, to apply the updates immediately:
+  
+* go to the Deploy tab 
+* in the manual deploy section deploy the master branch
```

Comment:
So this step doesn't appear to be universal. For a configured instance the step you detail applies
![screen shot 2017-08-16 at 14 32 52](https://user-images.githubusercontent.com/1039017/29365716-e4f10498-828f-11e7-9aa5-6a1f8bb8eadb.png)
![screen shot 2017-08-16 at 14 32 56](https://user-images.githubusercontent.com/1039017/29365719-e644c348-828f-11e7-9164-352994e9ce88.png)
However for an instance I set up using bothan-deploy and then didn't make any additional changes this is the default option
![screen shot 2017-08-16 at 14 33 49](https://user-images.githubusercontent.com/1039017/29365762-02b2fa4a-8290-11e7-93fe-3d2719285026.png)
So I just connected the account to github and enabled a manual deploy, works as the steps detail.
Should connecting to github for automatic deploys be included in the documentation?